wordpress-mobile/WordPress-iOS
# Custom Styles

Refs #12969

Depends on https://github.com/wordpress-mobile/WordPress-iOS/pull/13031

## To test

1. Add an image in Aztec
2. Tap on it then "Edit"
3. Check the style of the screen shown

PR submission checklist:
